### ISSUE TITLE
add support for periodicals

### ISSUE THREAD

**Issue by maxlath** [inventaire/inventaire#254]
### motivation
* there were [requests for supporting periodicals](https://trello.com/c/InrxOEIJ/107-periodical-and-comics-management) for a while now
* people [started adding them without waiting](https://inventaire.io/search?q=astrapi)
* and now, a small library gave us a dump of their data (some 2000 entries), in order to reconcile the entities data before importing them in there inventory; and that dump contains quite a lot of magazines

### now what
Would that be that difficult to add support for those periodicals? As I see it, we could "simply" create [a new entity meta-type](https://github.com/inventaire/inventaire/blob/d83ce2a/server/lib/wikidata/aliases.coffee#L6-L114), which would differ from `works` in one way: instead of the usual
```
inventory examplar (aka item) --> edition (wdt:P31-wd:Q3331189)`

**Comment by ivanhercaz** [comment]
Very, very interesting proposal @maxlath! What about with the scientific journals? I think it could be a very interesting type for inventaire, easy to add, at least in my mind... I admit that my interesting in this is specifically with the scientific scope because I have some journals with and without ISSN in my library.

**Comment by nclm** [comment]
There are cases where there might be several editions of the same issue of a periodical. For instance where it’s re-issued after the first batch runs out and the ISBN changes for the second edition, or when a omnibus/bounded version of several issues is published.

For instance (which is hackly marked as a book, part of a book series, but really it is an issue, part of a periodical publication): https://inventaire.io/entity/wd:Q110536578

It has three editions:
* Two editions of the single issue.
* One omnibus edition bound with other issues.

That considered, I’m wondering if there is a need for a different structure. For sure inventaire could support this structure if it’s used on Wikidata. But in the meantime:

* Periodicals (for instance of types “magazine”, “trade magazine”, and others types) could be displayed exactly like series.
* The property “issue” (P433) could be treated just as series ordinals.
* The issues themselves (Q28869365) could be shown as works. Most would have only have one edition.

Basically treating periodicals as a series of synonyms with books. Maybe just as a temporary solution? But it would be nice if periodicals could at least appear correctly on inventaire and be added without cheating in the “instance of”.

Another example: https://inventaire.io/entity/wd:Q123145932 (an issue which is marked as a “written work” to be added on inventaire), but the magazine itself https://inventaire.io/entity/wd:Q123145923 does not display as a series.

**Comment by maxlath** [comment]
There is a [WikiProject Periodicals](https://www.wikidata.org/wiki/Wikidata:WikiProject_Periodicals) from which we could draw a data model
